Documentation build fails

(#27939)
I tried to build the documentation using Docker, but I encountered the following error:

```bash
Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/sphinx/config.py", line 356, in eval_config_file
    exec(code, namespace)  # NoQA: S102
  File "/sympy/doc/src/conf.py", line 25, in <module>
    import sympy
  File "/sympy/sympy/__init__.py", line 18, in <module>
    raise ImportError("Python version 3.9 or above is required for SymPy.")
ImportError: Python version 3.9 or above is required for SymPy.

make[1]: *** [Makefile:74: printwarnings] Error 1
make: *** [Makefile:55: html] Error 2

```

It seems that the error is caused by the fact that the Docker environment uses Python 3.8, while SymPy now requires Python 3.9 or higher.
I suspect the issue comes from this line, but I'm not sure which version I should change it to:
https://github.com/sympy/sympy/blob/3a4a13236c07c1a025b857777fa7e69f10824b01/doc/Dockerfile.htmldoc#L57

--- a/.mailmap
+++ b/.mailmap
@@ -923,6 +923,7 @@ Lejla Metohajrova <l.metohajrova@gmail.com>
 Lennart Fricke <lennart@die-frickes.eu>
 Leo Battle <leowbattle@gmail.com>
 Leonardo Mangani <leomangani4@gmail.com> Leonardo Mangani <88255254+leomanga@users.noreply.github.com>
+Leonardo Mangani <leomangani4@gmail.com> leomanga <leomangani4@gmail.com>
 Leonid Blouvshtein <leonidbl91@gmail.com>
 Leonid Kovalev <leonidvkovalev@gmail.com> <leonid.traveling@gmail.com>
 Leonid Kovalev <leonidvkovalev@gmail.com> <normalhuman@users.noreply.github.com>

--- a/doc/Dockerfile.htmldoc
+++ b/doc/Dockerfile.htmldoc
@@ -54,7 +54,7 @@
 #
 ###############################################################################
 
-FROM python:3.8.12-slim-buster
+FROM python:3.13.3-slim-bookworm
 
 RUN apt-get update
 RUN apt-get install -y make librsvg2-bin imagemagick graphviz git

